packages/rum-core/test/performance-monitoring/transaction.spec.js

```diff
@@ -168,31 +125,6 @@ describe('transaction.Transaction', function() {
     expect(span).toBe(undefined)
   })
 
-  it('should not produce negative durations while adjusting to the spans', function() {
```

Comment:
Shouldn't we move this test to performance monitoring as well?

Reply:
Oh yeah might have missed, Let me add it